rpcserver.go

```diff
@@ -185,6 +185,10 @@ var (
 		},
 	}
 
+	// adminPermissions is a slice of all the entities that exist, giving
+	// a user full admin access to all RPCs.
+	adminPermissions = append(readPermissions, writePermissions...)
```

Comment:
not sure if this is safe, as `append` can actually modify the passed arguments 😱

Reply:
Hmm... I wasn't aware of this. Shouldn't we have way more unexplained bugs if this really was an issue in practice though?

Reply:
We did a sweep of the codebase at one point to fix the cases we had IIRC. Not sure how likely it is to happen, but we had a bug once because of this.